sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/ChatViewModel.kt

```diff
@@ -1201,8 +1201,8 @@ abstract class ChatViewModel<ARGS: NavArgs>(
             return
         }
 
-        payAttachmentJob = viewModelScope.launch(mainImmediate) {
-            submitSideEffect(ChatSideEffect.AlertConfirmPayAttachment {
+        submitSideEffect(ChatSideEffect.AlertConfirmPayAttachment {
```

Comment:
we should be launching a coroutine using viewModelScope here. The method does not need to be a suspend method and can simply be called from the click listener.